gensim/models/word2vec_inner.pyx

```diff
@@ -720,8 +721,8 @@ def train_batch_cbow(model, sentences, alpha, _work, _neu1, compute_loss):
                 if negative:
                     next_random = fast_sentence_cbow_neg(negative, cum_table, cum_table_len, codelens, neu1, syn0, syn1neg, size, indexes, _alpha, work, i, j, k, cbow_mean, next_random, word_locks, _compute_loss, &_running_training_loss)
 
-    model.running_training_loss = _running_training_loss
-    return effective_words
+    model.running_training_loss += _running_training_loss
+    return effective_words, effective_words
```

Comment:
It is worth writing a comment, why return same value twice

Reply:
I added the reason for that in the new docstring.